keps/sig-network/1880-multiple-service-cidrs/README.md

```diff
@@ -697,7 +682,7 @@ it will be safe to disable the dual-write mode.
 | 1.31     | Beta off | Alpha off   |
 | 1.32     | Beta on  | Alpha off   |
 | 1.33     | GA on    | Beta off   |
-| 1.34     | GA (there are no bitmaps running) | GA on (also delete old bitmap)|
+| 1.34     | GA (there are no bitmaps running) | GA (also delete old bitmap)|
 | 1.35     | remove feature gate | remove feature gate  |
 
 ## Production Readiness Review Questionnaire
```

Comment:
Missing elements in this section of the document:
1. In `###### How can this feature be enabled / disabled in a live cluster?` there's only `MultiCIDRServiceAllocator` described, you need to add `DisableAllocatorDualWrite` there too. 
2. Update Implementation History. 

A small nit, but the alternative section is a bunch of links, can you copy&paste those comments next to the links for ease of reading?

Reply:
done